doctr/__main__.py

```diff
@@ -89,8 +89,7 @@ def get_parser():
         public repositories for the user. This option is not recommended
         unless you are using a separate GitHub user for deploying.""")
     configure_parser.add_argument("--no-upload-key", action="store_false", default=True,
-        dest="upload_key", help="""Don't automatically upload the deploy key to GitHub. If you select this
-        option, you will not be prompted for your GitHub credentials. """)
+        dest="upload_key", help="""Don't automatically upload the deploy key to GitHub.""")
```

Comment:
Hmm... That's reasonable, but I think we should probably only do that in the `--no-upload-key` case, since we'll need the login info saved to pass to `GitHub_post` and `encrypt_variable` and a bunch of other functions otherwise.